service/azureconfig/v1/resource/service/desired.go

```diff
@@ -0,0 +1,46 @@
+package service
+
+import (
+	"context"
+
+	"k8s.io/api/core/v1"
+	apismetav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/util/intstr"
+
+	"github.com/giantswarm/microerror"
+
+	"github.com/giantswarm/azure-operator/service/azureconfig/v1/key"
+)
+
+func (r *Resource) GetDesiredState(ctx context.Context, obj interface{}) (interface{}, error) {
+	customObject, err := key.ToCustomObject(obj)
+	if err != nil {
+		return nil, microerror.Mask(err)
+	}
+
+	service := &v1.Service{
+		ObjectMeta: apismetav1.ObjectMeta{
+			Name:      "master",
+			Namespace: key.ClusterID(customObject),
+			Labels: map[string]string{
+				"app":      "master",
+				"cluster":  key.ClusterID(customObject),
+				"customer": key.ClusterCustomer(customObject),
```

Comment:
In format we agreed on other labels. Lets add the new style as well in case the others have to be maintained for now. See also https://github.com/giantswarm/fmt/.